### ISSUE TITLE
How to load the finetuned checkpoint for inference?

### ISSUE THREAD

**Issue by AndysonYs** [baaivision/NOVA#13]
Hi. After I finetuned a new model with the training script, I get the checkpoint files like below

transformer/
config.json  diffusion_pytorch_model.bin  

Could you please explain how I can use the checkpoint for inference?

**Comment by PhyscalX** [comment]
You can copy or create symlink for the missing pipeline components, except for ``transformer``:
- ``scheduler``
- ``text_encoder``
- ``tokenizer``
- ``vae``
- ``model_index``

**Comment by PhyscalX** [comment]
Storing and loading ``safetensors`` are trivial and not natively supported by PyTorch.

We discard this format in our developments.
